### PR TITLE
Update docs with multiple url usage

### PR DESCRIPTION
**Pull request by ryfu-msft**
- Adds additional information to the new command and update command documentation regarding how to utilize multiple urls.
- Also elaborates more on how multiple installer urls are matched in the update command.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-create/pull/109)

### CODE REVIEW

File: doc/new.md [hunk 1]
```diff
@@ -4,9 +4,13 @@ The **new** command of the [Winget-Create](../README.md) tool begins the process
 
 ## Usage
```
Comment (palenshus) [inline doc/new.md:5]: readme.md needs to be update too:

> The CLI only supports a single installer URL. If you have a more complex manifest with multiple installer URLs, you'll need to update the manifests manually, and then you can use wingetcreate submit to submit the updated manifests to GitHub.

File: doc/new.md [hunk 2]
```diff
@@ -4,9 +4,13 @@ The **new** command of the [Winget-Create](../README.md) tool begins the process
...
 
-`WingetCreateCLI.exe new [<url>] [\<options>]`
+`WingetCreateCLI.exe new [<urls>] [\<options>]`
 
-The **new** command can be called with the optional URL.  If the URL is provided, **Winget-Create** will download the installer as it begins.  If the URL is not included, the user will need to add it when prompted.
+The **new** command can be called with the optional URL(s). **Winget-Create** will download each provided installer as it begins.  If the URL is not included, the user will need to add it when prompted. When prompted, use a comma to separate each installer url you wish to include in your generated manifest.
```
Comment (palenshus) [inline doc/new.md:9]: It's unfortunate that when multiple urls are provided on the command line, they're space delimited, which is different than described below. Can you incorporate that into the example on line 7?

File: doc/update.md [hunk 3]
```diff
@@ -25,18 +44,18 @@ The following arguments are available:
...
 2) call the WingetCreateCLI.exe
 
-`WingetCreateCLI.exe update <PackageIdentifier> --url <InstallerUrl> --token <token> --version <version>`
+`WingetCreateCLI.exe update <PackageIdentifier> --urls <InstallerUrls> --token <token> --version <version>`
 
 ### PackageIdentifier  
 
 The first action **Winget-Create** will do is to download the metadata associated with the **PackageIdentifier**.
 
 **Winget-Create** will then replace the following manifest values with the values provided on the command line:
```
Comment (palenshus) [inline doc/update.md:56]: This list is pretty limited compared to what we do now. I might just remove it altogether, as it'll be hard to keep it up to date.

**Review by palenshus — CHANGES_REQUESTED** [review]

File: doc/new.md (hunk 1 above)
Comment (ryfu-msft) [reply doc/new.md:5]: Updated readme

File: doc/new.md (hunk 2 above)
Comment (ryfu-msft) [reply doc/new.md]: Added an extra example for the command line scenario.

File: doc/update.md (hunk 3 above)
Comment (ryfu-msft) [reply doc/update.md]: I agree. I decided to remove both the "Winget-Create update Command flow" and the "PackageIdentifier" sections as they're not particularly useful and out of date. There's better information about using wingetcreate in a CI/CD scenario in the readme doc.

File: doc/update.md [hunk 4]
```diff
@@ -42,24 +42,9 @@ The following arguments are available:
...
-The first action **Winget-Create** will do is to download the metadata associated with the **PackageIdentifier**.
-
-**Winget-Create** will then replace the following manifest values with the values provided on the command line:
-* Version
-* Urls
-
-Finally, **Winget-Create** will calculate and update the hash.
-
-If you have provided your [GitHub token](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token) on the command line along with the **--submit** command and the device is registered with Github, **Winget-Create** will submit your PR to [Windows Package Manager repo](https://docs.microsoft.com/en-us/windows/package-manager/).  
+If you have provided your [GitHub token](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token) on the command line along with the **--submit** command and the device is registered with Github, **Winget-Create** will automatically submit your PR to [Windows Package Manager repo](https://docs.microsoft.com/en-us/windows/package-manager/).
```
Comment (palenshus) [inline doc/update.md:47]: What does "the device is registered with Github" mean?

**Review by palenshus — CHANGES_REQUESTED** [review]

File: doc/update.md (hunk 4 above)
Comment (ryfu-msft) [reply doc/update.md]: This is inaccurate and does not apply to submitting with a token. I removed the statement

**Review by palenshus — APPROVED** [review]

**Pull request merged**